### ISSUE TITLE
When there are errors in the file being edited and I press ctrl + shift + m, orca announces only the following: tree

### ISSUE THREAD

**Issue by jvesouza** [microsoft/vscode#95516]
Issue Type: <b>Bug</b>

1. Launch orca.
2. Create an empty java script file.
3. Insert the following line in the file.
lvar x
4. Press ctrl+shift+m.

Orca should announce the existence of errors but instead announces only the following: tree.

CC @isidorn

VS Code version: Code - Insiders 1.45.0-insider (a3083e58e2a9e77f60e531acaa32e18527ae8439, 2020-04-17T08:09:45.292Z)
OS version: Linux x64 5.6.4-arch1-1

**Comment by isidorn** [comment]
@jvesouza thanks for filling this issue. I agree it makes sense.
The issue here is that when we create a Markers tree we do not see its global aria label [here](https://github.com/microsoft/vscode/blob/aecef14e24b4c3643bb6025c75819be4a608be1f/src/vs/workbench/contrib/markers/browser/markersView.ts#L407).
However we would need a mechanism to update the `ariaLabel` of the tree, since the `ariaLabel` depends on the number of errors and is dynamic.

@joaomoreno is it possible to add some list / tree api for this? Alternative is to do a hack and manually set this on the HTML container of the tree.

**Comment by jvesouza** [comment]
When the file has no errors a beautiful information appears saying that there are no errors. I believe that in the case of the existence of errors it would be enough to show information saying that there are errors.
Something like this: problems detected.

**Comment by isidorn** [comment]
That is because in that case there is not tree, and @sandy081 nicely sets the `aria-labelledby` property.
The tree does not support this `aria-labelledby`. So that would be an alternative way to solve this.

**Comment by isidorn** [comment]
After discussing with Joao we will add setters and getters for `list.ariaLabel`.
I can look into this.

**Comment by jvesouza** [comment]
There seems to be similar problem when using the Focus Breadcrumbs command.
When I press ctrl + shift +. the orca announces:
tree.

**Comment by isidorn** [comment]
I have tackled the Problems view issue and the Breadcrumbs issue via this PR https://github.com/microsoft/vscode/pull/95877

Will merge it later today / tomorrow so should be soon in insders
@jvesouza thanks for filling great bugs!

**Comment by jvesouza** [comment]
@isidorn  It was fixed and it looked great. 
Thank you for the excellent work, correcting bugs and implementing new features.

**Comment by isidorn** [comment]
@jvesouza thanks for letting us know it is good now. Adding verified label.
